tools/functional-tester/etcd-tester/tester.go

```diff
@@ -110,7 +110,8 @@ func (tt *tester) runLoop() {
 				}
 			}
 			if !ok {
-				log.Printf("etcd-tester: [round#%d case#%d] checking current revisions failure...", i, j)
+				time.Sleep(time.Second)
```

Comment:
@gyuho Actually check https://github.com/coreos/etcd/pull/4449#discussion_r52110943. I think we do not even need to fix agent.